lib/game.js

```diff
@@ -0,0 +1,52 @@
+function Game() {
+  this.frames = [];
+  MAX_FRAMES = 10;
+};
+
+Game.prototype = {
+  getFrames: function () {
+    return this.frames
+  },
+
+  currentFrame: function() {
+    return this.frames[this.frames.length - 1];
+  },
+
+  currentFrameNo: function () {
+    return this.frames.length
+  },
+
+  addFrame: function (frame) {
```

Comment:
Adding frames requires something outside to know about the internal implementation of game. I would prefer to inject an array of frames at the start of the game and have the game manage which one is the currently active frame as the game develops. The way you've done this creates a coupling between game and whatever is calling it. 

Injecting an array of frames also means you can initialise your game in any state you choose, which can help with test setup.

Reply:
`addFrame` was actually only ever called by the game itself, so it was effectively a private method (just not marked as such). The outside world called addRoll and then the game decided whether it needed a new frame or not. However, I see your point re injecting an array of empty frames making testing much easier, so I've updated this